### ISSUE TITLE
Question/suggestion, on leave per site.

### ISSUE THREAD

**Issue by zippydsmlee** [Lusito/forget-me-not#213]
On leave seems to remove cookies when I leave any site and not the site in question. Is that how it's supposed to work in Firefox/water fox?

**Comment by Lusito** [comment]
No, it's not supposed to work like that. I don't have Waterfox and I'm not sure how compatible this extension is with Waterfox.

I can only imagine that your configuration is incorrect. If you share your config, maybe I can assist.
